CHANGELOG.md

```diff
@@ -24,8 +32,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Change (unused yet) `config.giantswarm.io/major-version` annotation to `config.giantswarm.io/version`.
 
-## [3.6.0] - 2020-12-01
```

Comment:
Did something go wrong here?

Reply:
A bad revert perhaps. I'll bring it back.